project/AkkaDisciplinePlugin.scala

```diff
@@ -66,14 +66,15 @@ object AkkaDisciplinePlugin extends AutoPlugin {
     "akka-testkit")
 
   // cat=lint-deprecation: we want to keep using both Java and Scala deprecation annotations
-  val defaultScala2Options = "-Wconf:cat=unused-nowarn:s,cat=lint-infer-any:s,cat=lint-deprecation:s,any:e"
+  val defaultScala2Options =
+    "-Wconf:cat=unused-nowarn:s,cat=lint-infer-any:s,cat=lint-deprecation:s,msg=Use Akka Distributed Cluster:s,any:e"
```

Comment:
Strange, this is supposed to work, but it is not?
https://www.scala-lang.org/2021/01/12/configuring-and-suppressing-warnings.html